packages/hawtio/src/plugins/springboot/TraceView.tsx

```diff
@@ -0,0 +1,328 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import {
+  Bullseye,
+  Button,
+  CodeBlock,
+  CodeBlockCode,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Flex,
+  FormGroup,
+  Label,
+  Modal,
+  PageSection,
+  Pagination,
+  SearchInput,
+  Toolbar,
+  ToolbarContent,
+  ToolbarFilter,
+  ToolbarGroup,
+  ToolbarItem,
+} from '@patternfly/react-core'
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table'
+import { CheckCircleIcon, ExclamationCircleIcon, SearchIcon } from '@patternfly/react-icons'
+import { Trace } from './types'
+import { springbootService } from './springboot-service'
+
+const HttpStatusIcon: React.FunctionComponent<{ code: number }> = ({ code }) => {
+  if (code < 400) return <CheckCircleIcon color={'#3E8635'} />
```

Comment:
why need `{'...'}` instead of just `'...'`?

Reply:
Probably it's default in my IDE. btw I've noticed if it's in `{}` I have also displayed used color directly in IDE: 

<img width="597" alt="Screenshot 2023-11-23 at 10 53 17" src="https://github.com/hawtio/hawtio-next/assets/6814482/0338ac07-62ff-41bf-9b56-cd9838890532">

Reply:
Maybe it's better to use PF CSS variables then, but it's ok for now.